tests/test_configs.py

```
@@ -0,0 +1,32 @@
+import json
+from pathlib import Path
+import unittest
+
+
+class TestDockerConfigs(unittest.TestCase):
+    def _load_config(self, path: str):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_load_config`

Add return type annotation: `None`

(ANN202)